README.md

```diff
@@ -1,2 +1,59 @@
-# ProjectReviewExample
-A dummy project to learn code review process by Microverse.
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# My YouTube Clone Project with Microverse
+
+> Description the project.
+
+## Built With
+
+- HTML
+- CSS
+- Git
+
+## Live Demo (if available)
+
+[Live Demo Link](https://github.com/egichuhis/)
+
```

Comment:
Kindly consider adding a valid link to the live demo for this project or deleting this section entirely if the link is non-existent.